packages/wallet/frontend/src/ui/forms/Checkbox.tsx

```diff
@@ -0,0 +1,36 @@
+import { forwardRef, useId } from 'react'
+import type { ComponentPropsWithoutRef } from 'react'
+import { FieldError } from './FieldError'
+
+type CheckboxProps = ComponentPropsWithoutRef<'input'> & {
+  label: string
+  error?: string
+}
+
+export const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
+  ({ error, label, ...props }, ref) => {
+    const id = useId()
+    return (
+      <div>
+        <div className="mt-1 flex justify-evenly">
+          <input
+            id={id}
+            ref={ref}
+            type="checkbox"
+            className="mt-1 cursor-pointer rounded-md border border-turqoise text-green-3 focus:ring-0 disabled:bg-gray-50"
```

Comment:
Let's not remove the ring. It should just have another color (for accessibility, which is on our list to improve keyboard navigation).